django/contrib/gis/db/models/sql/conversion.py

```diff
@@ -24,6 +24,9 @@ def __init__(self, area_att):
         self.area_att = area_att
 
     def from_db_value(self, value, expression, connection, context):
+        if connection.ops.oracle and value == u'':
+            # Oracle returns empty string for a strange reason
```

Comment:
Do you think this is related to `connection.features.interprets_empty_strings_as_nulls`?